content/swagger/aep/public-oas.yml

```
@@ -178,7 +178,7 @@ paths:
             type : string
       responses:
         '200':
-          description: Subscriptions successfully retrieved
+          description: Returns the list of subscriptions that are either active or revoked.
```

can we update in event but also here the comment :pray: 
```suggestion
          description: Returns the list of subscriptions that are either active and suspended.
```

---

I've had a look at the code and there are 3 statuses available for this endpoint (active, suspended, revoked).
I'll fix it (here and on the AEP repo)